app/app.go

```diff
@@ -297,6 +297,21 @@ func NewOsmosisApp(
 	)
 	app.UpgradeKeeper = upgradekeeper.NewKeeper(skipUpgradeHeights, keys[upgradetypes.StoreKey], appCodec, homePath)
 
+	// this configures a no-op upgrade handler for the "upgrade-lockup-module-store-management" upgrade
+	app.UpgradeKeeper.SetUpgradeHandler("upgrade-lockup-module-store-management", func(ctx sdk.Context, plan upgradetypes.Plan) {
+		locks, err := app.LockupKeeper.GetPeriodLocks(ctx)
```

Comment:
This approach looks right to me, just want to double check via a test that there isn't some edge case we're missing

Reply:
Added test